index.js

```diff
@@ -35,41 +35,41 @@ QUnit.notifications = function(options) {
           }
         }
 
-        var notification = new window.Notification(title, _options);
+        var notifications = new window.Notification(title, _options);
```

Comment:
Simply for concistency sake:

- "Hide passed tests" => `hidepassed`
- "Check for Globals" => `noglobals`
- "No try-catch" => `notrycatch`
- "Notifications" => `notifications`